src/components/Dropdown.tsx

```diff
@@ -0,0 +1,109 @@
+import React, {
+  useCallback,
+  useMemo,
+  useState,
+  useRef,
+  useEffect,
+} from 'react';
+import debounce from 'lodash.debounce';
+import classNames from 'classnames';
+import { Person } from '../types/Person';
+
+interface Props {
+  people: Person[];
+  delay: number;
+  onSelected?: (user: Person) => void,
+}
+
+export const Dropdown: React.FC<Props> = ({
```

Comment:
https://github.com/mate-academy/react_autocomplete/pull/238#discussion_r1324816582
Dropdown is a common name for a component that has a dropdown list.

Therefore, I agree with Alexander that it is worth giving a more personalized name or combining both options: `DropdownAutocomplete`, for example.
<img width="1431" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/54860641/c8d74758-5eea-4985-8dee-60ce9728a3b5">

Also, you can see the approach of the developers of the MUI library, that they would also call such a component `Autocomplete` :)

<img width="1168" alt="image" src="https://github.com/mate-academy/react_autocomplete/assets/54860641/ee5910e6-d506-41da-9bee-d56380d4fccf">

Reply:
:( ok